src/python/director/cameracontrolpanel.py

```diff
@@ -5,6 +5,7 @@
 from director import cameracontrol
 from director import propertyset
 from director import pointpicker
+from director import roboturdf
```

Comment:
unfortunately this file should not import roboturdf, it's new dependency that is too big for the cameracontrolpanel.py.  The camera control panel was designed to be used as a basic component, shouldn't have robot dependencies.  I will make a new pull request to remove this dependency but keeps your new feature.